doc/nrf/protocols/matter/end_product/factory_provisioning.rst

```diff
@@ -13,6 +13,937 @@ The factory data parameter set includes different types of information, for exam
+   [*]     Enable generation of a new Rotating device id unique id
+   (11223344556677889900) Serial number of device
+   (Nordic Semiconductor ASA) Human-readable vendor name
+   (not-specified) Human-readable product name
+   (2022-01-01) Manufacturing date in ISO 8601
+   (0) Integer representation of hardware version
+   (prerelease) user-friendly string representation of hardware ver
+   (0xF00) Device pairing discriminator
+   (20202021) SPAKE2+ passcode
+   (1000) SPAKE2+ iteration count
+   (U1BBS0UyUCBLZXkgU2FsdA==) SPAKE2+ salt in string format
+   (uWFwqugDNGiEck/po7KHwwMwwqZgN10XuyBajPGuyzUEV/iree4lOrao5GuwnlQ
+   (91a9c12a7c80700a31ddcfa7fce63e44) A rotating device id unique i
+
+Default Kconfig values and developing aspects
+=============================================
+
+Each factory data parameter has its default value reflected in the Kconfig.
+The list below shows some Kconfig settings that are configured in the nRF Connect build system and have an impact on the application.
+You can modify them to achieve the desired behavior of your application.
+
+* The device uses the test certificates located in the :file:`credentials/development/attestation/` directory, which are generated using all default values.
+  If you want to change the default ``vendor_id``, ``product_id``, ``vendor_name``, or ``device_name`` and generate new test certificates, set the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_CERT_SOURCE_GENERATED` Kconfig option to `` y``.
+  Remember to build the ``chip-cert`` application and add it to the system PATH.
+
+  For developing a production-ready device, you need to write the certificates obtained during the certification process.
+  Set the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_CERT_SOURCE_USER` Kconfig option to ``y`` and set the appropriate paths for the following Kconfig options:
+
+  * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_USER_CERTS_DAC_CERT`
+  * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_USER_CERTS_DAC_KEY`
+  * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_USER_CERTS_PAI_CERT`
+
+* By default, the SPAKE2+ verifier is generated during each example's build.
+  This means that this value will change automatically if you change any of the following parameters:
+
+  * :kconfig:option:`CONFIG_CHIP_DEVICE_SPAKE2_PASSCODE`
+  * :kconfig:option:`CONFIG_CHIP_DEVICE_SPAKE2_SALT`
+  * :kconfig:option:`CONFIG_CHIP_DEVICE_SPAKE2_IT`
+
+  You can disable the generation of the SPAKE2+ verifier by setting the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_GENERATE_SPAKE2_VERIFIER` Kconfig option to ``n``.
+  You also need to provide the externally-generated SPAKE2+ verifier using the :kconfig:option:`CONFIG_CHIP_DEVICE_SPAKE2_TEST_VERIFIER` Kconfig value.
+
+* Generating the rotating device ID unique ID is disabled by default, but you can enable it by setting the :kconfig:option:`CONFIG_CHIP_ROTATING_DEVICE_ID` and :kconfig:option:`CONFIG_CHIP_DEVICE_GENERATE_ROTATING_DEVICE_UID` Kconfig options to ``y``.
```

Comment:
... "the rotating device ID unique ID": could this be "the unique rotating device ID"?

Reply:
It should be unique ID for rotating device ID. These are two separate ids, and one is required by the other one.